### DISCUSSION TITLE
/v6/deployments/{id}/files returning empty array

### DISCUSSION THREAD

**Discussion by CalinaCristian** [vercel/vercel#8122] [Help]
Hello!

Based on [this documentation](https://vercel.com/docs/rest-api#endpoints/deployments/list-deployment-files), the endpoint should've returned a list of all the files that you have in a deployment.

This used to work fine, since I had a [package](https://github.com/CalinaCristian/source-from-vercel-deployment) for a long time (since zeit) that used this endpoint to recreate locally what it finds on this API endpoint. 

I see that when going to my deployment on vercel, this endpoint is not used to recreate the file structure.
Another endpoint (which is not even in the API documentation) is used to fetch the files, and then each individual file is requested on separate API calls -- I used this one as well in an update to my package to make it work.

![image](https://user-images.githubusercontent.com/9513641/178119917-cca7d66b-324e-4536-93e2-b10c7eaa5eee.png)
![image](https://user-images.githubusercontent.com/9513641/178119923-5bd8b14c-2fbd-4440-8c8a-27c96b747ff7.png)

Is the `/v6/deployments/{id}/files` deprecated or what is the issue with it that it returns an empty array even if I request from a team or from personal projects?

(I am getting 200 response with an empty array on the data)

![image](https://user-images.githubusercontent.com/9513641/178120058-f5ccce43-fee8-44ad-a7b3-ad8604754e1d.png)
![image](https://user-images.githubusercontent.com/9513641/178120063-dded551b-de87-4e60-8cbf-e8baee160e7e.png)

**Comment by amyegan** [comment]
Hi @CalinaCristian. The [List Deployment Files](https://vercel.com/docs/rest-api#endpoints/deployments/list-deployment-files) endpoint `/v6/deployments/{id}/files` should still be working, but I can confirm I'm getting the same empty array result that you are. We're looking into what's causing the problem and I'll update here once I know more.

In the meantime, can you tell me when the project was last deployed? I would guess it was in the last few days, but I don't want to assume. This timing could help narrow down the cause.

**Reply by CalinaCristian** [reply]
Yes, it was deployed in one day before I posted. I was just testing my package and saw that it didn't work anymore.
